config/set/coding-style.php

```diff
@@ -69,6 +68,5 @@
 
     $services->set(CamelCaseFunctionNamingToUnderscoreRector::class);
     $services->set(SplitGroupedUseImportsRector::class);
-    $services->set(UnderscoreToCamelCaseVariableNameRector::class);
```

Comment:
Hmmmm..., @TomasVotruba should I keep it in both Sets?

Reply:
That would be confusing I guess. Better have it unique.

Reply:
Ready for merging then.